app/assets/javascripts/host_edit.js

```diff
@@ -166,6 +166,7 @@ function submit_with_all_params() {
     },
     error: function(response) {
       $('#content').html(response.responseText);
+      $(document.body).trigger('ContentLoad');
```

Comment:
Shouldn't we move it to the `complete` then?

Reply:
make sense :+1:  and also remove it from `success` callback